Bottom: a0e9b5dc9ad3eb5b74e578245b8a85cae9600d8d
Top:    e736b9f008c143a6c15f87f37068573f64082250
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-07 01:46:11 +0000

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index 1ef631b..3bd66f6 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3093,7 +3093,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
